microservices/Order.py

```diff
@@ -0,0 +1,132 @@
+from flasgger import swag_from
+from flask import Blueprint, request, session
+from flask_login import login_required, login_user, logout_user
+from werkzeug.security import check_password_hash
+from flask import Blueprint, request, make_response, jsonify
+from flask.views import MethodView
+from project.server import bcrypt, db
+from project.server.models import User, BlacklistToken
+
+@app.route('/')
+@app.route('/order/')
+def showorders():
+    # query db and assign to orders variable
+    orders = session.query(order).order_by(order.id.asc()).all()
+
+    # Check if user credential match data in login_session
+    # Check if username data in login_session
+    if 'username' not in login_session:
+        # This page will redirect user to Google login page
+        return render_template('publicorders.html', orders = orders, title = "orders")
+    else:
+        # return "This page will show all orders"
+        return render_template('orders.html', orders = orders, title = "orders")
+
+
+@app.route('/order/new/', methods = ['GET', 'POST'])
+# Method to create neworder
+def neworder():
+    # Check to see if a user is logged in.
+    if 'username' not in login_session:
+        return redirect('/login')
+
+    # If user is logged in then access neworder page
+    if request.method == 'POST':
+        # Store order input from form into neworder
+        neworder = order(
+            name = request.form['name'].strip(),
+            street = request.form['street'].strip(),
+            city = request.form['city'].strip(),
+            state = request.form['state'].strip(),
+            zipCode = request.form['zipCode'].strip(),
+            phone = request.form['phone'].strip(),
+            email = request.form['email'].strip(),
+            website = request.form['website'].strip(),
+            cuisine = request.form['cuisine'].strip(),
+            description = request.form['description'].strip(),
+            delivery = request.form['delivery'].strip(),
+            user_id = login_session['user_id']
+        )
+        # add neworder item to db stage
+        session.add(neworder)        # commit neworder item to db
+        session.commit()
+
+        # flash message to indicate success
+        flash("New order: " + neworder.name + " ==> Created!")
+        # redirect user to updated list of orders
+        return redirect(url_for('showorders'))
+    else:
+        return render_template('neworder.html', title = "New order Input")
+
+# Edit a order
+#########################
+@app.route('/order/order id/edit/', methods = ['GET', 'POST'])

```

Comment:
should just be `@app.route('/order/{orderId}/edit/` and this should be only a POST request